### PR TITLE
[FIX] website: tour fails in multi-company (like l10n)

### PR DESCRIPTION
**Pull request by gawa-odoo**
When l10n_ar_website_sale is installed, the tour is ran. 
We have several companies, so the search is not selective enough. 
Just search on the right websites.

runbot-55875

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by gawa-odoo** [comment]
The error happens in 17.2+ (didn't check to see why there was a diff), but I don't see why we wouldn't want this in 17.1 already.

**Comment by gawa-odoo** [comment]
Nvm, it did happen before, didn't see the error :smile: .
Rebased to 16.0

**Comment by gawa-odoo** [comment]
@odoo/rd-website up?

File: addons/website/tests/test_page_manager.py [hunk 1]
```diff
@@ -21,6 +21,6 @@ def test_01_page_manager(self):
         self.start_tour(url, 'website_page_manager', login="admin")
         self.start_tour(url, 'website_page_manager_session_forced', login="admin")
 
-        alternate_website = self.env['website'].search([], limit=2)[1]
+        alternate_website = self.env['website'].search([('company_id', '=', self.env.company.id)], limit=2)[1]
```
Comment (rdeodoo) [inline addons/website/tests/test_page_manager.py:24]:
```suggestion
        alternate_website = self.env['website'].search([('name', '=', 'My Website 2')], limit=2)[1]
```
Let's go with this, searching explicitly what we are looking for. Code was not ideal before indeed :grimacing: but let's not go with another "indirect" way (could still break later) of searching the website

File: addons/website/tests/test_page_manager.py [hunk 2]
```diff
@@ -21,6 +21,6 @@ def test_01_page_manager(self):
         self.start_tour(url, 'website_page_manager', login="admin")
```
Comment (rdeodoo) [inline addons/website/tests/test_page_manager.py:21]: About the commit msg, you can go with
```diff
- Linked to runbot error 55875
+ runbot-55875
```
Same notation as for `task-123` and `opw-123`

**Review by rdeodoo — CHANGES_REQUESTED** [review]
Thanks for the PR, quick change then LGTM.

> up?

It's more a ping than a up, we reply only when ping explicitly or if we see something bad at first sight through code owner, otherwise we just scroll quickly to be sure it's "ok".
Reason is that we got many PR pings through code owner, and most of the time there is still review from other teams to be done before we are asked to check (if a check is needed at all).

tl;dr: you can ping us directly if you need a review, otherwise we just scroll the diff to see if something is really wrong, and if nothing is wrong, we just do nothing apart from having noticed the diff

**Comment by gawa-odoo** [comment]
Sure, sorry, didn't know your process.
Thanks

**Review by rdeodoo — APPROVED** [review]
@robodoo r+

Thank you
